Tests/Engine/Settings.tests.ps1

```diff
@@ -39,7 +39,7 @@ Describe "Settings Class" {
 
         'IncludeRules', 'ExcludeRules', 'Severity', 'RuleArguments' | ForEach-Object {
             It ("Should return empty {0} property" -f $_) {
-                $settings.${$_}.Count | Should Be 0
+                $settings.${$_}.Count | Should -Be 0
```

Comment:
there's a problem with `${$_}` which is not quite right as it is referencing the variable literally named `$_`, let's use the test case functionality of pester:
```
It "Should return empty <name> property" -TestCases @(
     @{ Name = "IncludeRules" }
     @{ Name = "ExcludeRules" }
     @{ Name = "Severity" }
     @{ Name = "RuleArguments" }
) {
${settings}.${Name}.Count | Should -Be 0
}
```

Reply:
Well spotted. When applying the syntax changes, I did not pay much attention to the test cases themselves but I can certainly improve this one.